src/help.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import Card from "./Components/Card";
+import { Link } from "react-router-dom";
+
+interface PageType {
+  imgUrl: string;
+  cardTitle: string;
+  cardBody: string;
+}
+
+// enum
```

Comment:
This is not an enum anymore. It is just an object. 

Since you use TS, you can use TS enums directly when needed.
https://www.typescriptlang.org/docs/handbook/enums.html